UnityProject/Assets/Scripts/Health/Living/HealthStateMonitor.cs

```diff
@@ -234,8 +248,17 @@ void SendBloodUpdate(GameObject requestor)
 
 	void SendRespiratoryUpdate(GameObject requestor)
 	{
-		HealthRespiratoryMessage.Send(requestor, gameObject, livingHealthBehaviour.respiratorySystem.IsBreathing,
-			livingHealthBehaviour.respiratorySystem.IsSuffocating);
+		HealthRespiratoryMessage.Send(requestor, gameObject, isSuffocatingCache);
+	}
+
+	void SendTemperatureUpdate(GameObject requestor)
```

Comment:
Is there a situation when client needs UI readings for another player?
I don't think so, so all cases of separate requestor and subject could be merged.
then it would be just `SendTemperatureUpdate()`
`HealthTemperatureMessage.Send( this.gameObject, temperatureCache );`
and message code itself would require neither `public NetworkInstanceId EntityToUpdate;` field,
nor SendToAll() method
because I just can't come up with a use case for that extra arg/field